deploy/install/01-prerequisite/03-crd.yaml

```diff
@@ -16,6 +16,39 @@ spec:
   scope: Namespaced
   versions:
   - name: v1beta1
+    served: true
```

Comment:
@jenting Can `controller-gen` in your PR generate this kind of old version info?

Reply:
Do you mean for v1beta1, still uses the old CRD schema that with `x-kubernetes-preserve-unknown-fields: true`, but for new v1beta2 uses the new CRD structural schema?

Reply:
I mean, by using `controller-gen` in your PR, will the following v1beta1 info be reserved/generated in the v1beta2 CRD YAML file?
https://github.com/longhorn/longhorn-manager/pull/1126/commits/86eabc9a11cfde7ca46e6610248fb4d98da882ec

Reply:
@shuo-wu @jenting 
I think this is just handled in my 2nd commit, please help check it. Thank you.

Reply:
Is file `deploy/install/01-prerequisite/03-crd.yaml` in the 2nd commit automatically generated by the script `k8s/generate_code.sh`? I saw it included both v1beta1 and v1beta2. If YES, that would be great.

Reply:
@shuo-wu Yes, now the all lines in `03-crd.yaml` are generated by `k8s/generate_code.sh`.